1-basics/package.json

```diff
@@ -3,29 +3,15 @@
   "version": "0.1.0",
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1"
+    "react-dom": "^18.2.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject"
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
```

Comment:
I would remove the script is you are not planning to teach them what's the preview.

My approach is always to keep the code base as minimal as possible for optimising the chance they feel understanding and controlling the code they see.